raiden-ts/src/config.ts

```diff
@@ -65,7 +68,8 @@ export const RaidenConfig = t.readonly(
       expiryFactor: t.number, // must be > 1.0
       httpTimeout: t.number,
       discoveryRoom: t.union([t.string, t.null]),
-      pfs: t.union([t.readonlyArray(t.union([Address, t.string])), t.boolean]),
+      additionalServices: t.readonlyArray(t.union([Address, t.string])),
```

Comment:
Can this also include monitoring services? :thinking: This makes no sense doesn't it? But eventually when ditch Matrix completely, we also wanna configure other services? So maybe make it explicit now and name it `additionalPfsServices` so no matter what comes later we are more stable for future extensions.

Reply:
it can include monitoring services. While MS doesn't expose an explicit API, it can use the same account as the PFS (it actually is a bundle) and we need to fetch the `address` for which to send `messageServiceSend` to `service: MS`. That's why it was named generically, because it can serve both, the registry is for both and the URL provides the address of the account for both (the difference is in the `deviceId`)

Reply:
Ah okay. Interesting. So then this configuration split change is actually even more beneficial. :+1: